opencv/src/test/scala/com/microsoft/azure/synapse/ml/opencv/ImageTransformerSuite.scala

```diff
@@ -172,6 +173,10 @@ class ImageTransformerSuite extends TransformerFuzzing[ImageTransformer] with Op
   lazy val images: DataFrame = spark.read.image.option("dropInvalid", value = true)
     .load(FileUtilities.join(fileLocation, "**").toString)
 
+  lazy val badImages: DataFrame =
+    spark.read.image.load(
+      ".//opencv//src//test//scala//com//microsoft//azure//synapse//ml//opencv//cmyk_image.jpeg")
```

Comment:
can we put this in datasets or use a smaller image here?

Reply:
we can put it in datasets, but you are the keymaster for making the dataset packages  :)

Reply:
if you want me to do it, you'll have to share the process

Reply:
nvm, I found a 2k image that works for the test instead of 1MB